conf/v4.routes

```diff
@@ -0,0 +1,2 @@
+GET /:nino/:taxYearClaimedFor/loss-claims   api.endpoints.lossClaim.list.v4.ListLossClaimsController.list(nino: String, taxYearClaimedFor: String, typeOfLoss: Option[String], businessId: Option[String], typeOfClaim: Option[String])
+GET /:nino/loss-claims                      api.controllers.DeprecatedController.version4(nino: String)
```

Comment:
This prevents a request with a V4 header and a V3 List Loss Claims Path from being routed to the V3 implementation, instead return a 404.